### PR TITLE
Deus ex working edition

### PR DESCRIPTION
**Pull request by gbasood**
Fixed the proc call on liver reagent metabolization that caused lifecode to runtime and break EVERYTHING from welding helmets to not being able to die.

### CODE REVIEW

**Comment by gbasood** [comment]
Tested on a local server, works as intended.
Just needs someone to merge it. If you want to see what I changed to make it work, 3cd312f

**Comment by LordMyroc** [comment]
Don't forget to add a FixOVein and Bone Gel to robotics so they can actually install the new lungs/eyes without having to steal tools from medbay.

**Comment by gbasood** [comment]
@LordMyroc If that's actually a problem I'm sure someone else can add it
I don't see a problem with it, robotics can just give the implants to the patients and they can go to medical for their elective surgery

File: html/changelogs/N3X15-deus-ex.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+author: Sood
```
Comment (ghost) [inline html/changelogs/N3X15-deus-ex.yml]: No need for the changelog, it's still in the game.

**Comment by LordMyroc** [comment]
I don't see how it's not a problem. It's already robotics' job to install, maintain and repair robotics limbs, and cyberorgans aren't really any different. Why should this explicitly robotics-related thing not be able to be done by the actual roboticists?

**Comment by gbasood** [comment]
Because robotic limbs use welding tools to fix, robotic organs require intensive surgery to insert/replace

**Comment by LordMyroc** [comment]
Robotic limbs ALSO requires intensive surgery to install/replace. They only difference is that they're easier to repair than internal organs.

**Comment by gbasood** [comment]
Exactly.

**Comment by LordMyroc** [comment]
So it's deferred to medical simply because it's not a job that can be done by any assistant who's ever been to primary tool storage?

**Comment by gbasood** [comment]
Because you have to do **ACTUAL SURGERY** not just chop off someone's limb and slap a robo arm on there.

It's not the job of this PR to define what a surgeon's job is, or what a roboticist's job is, so I'm not going to edit every map in rotation to give roboticists more surgery tools. If you want, make a feature request for it.

**Comment by LordMyroc** [comment]
Roboticists perform **ACTUAL SURGERY** all the time, how do you think we get the brains into their MMIs?

But fair enough, I'll make a feature request later on, since I assume this PR is mainly to fix the absolutely gamebreaking bugs and is supposed to be a quick merge.

**Comment by PJB3005** [comment]
inb4 fucks up again

**Comment by gbasood** [comment]
IT WON'T
If it does I'll give all my pomfcoins to charity

**Pull request merged**